src/tests/ftest/util/command_utils.py

```diff
@@ -625,8 +645,24 @@ def get_config_value(self, name):
         value = None
         if isinstance(self.yaml, YamlParameters):
             value = self.yaml.get_value(name)
+
         return value
 
+    def _get_result(self):
+        """Generate the yaml config if defined, then call the parent method.
+
+        Returns:
+            CmdResult: an avocado CmdResult object containing the command
+                information, e.g. exit status, stdout, stderr, etc.
+
+        Raises:
+            CommandFailure: if the command fails.
+
+        """
+        if self.yaml:
+            self.create_yaml_file()
+
+        return super(YamlCommand, self)._get_result()
```

Comment:
This shouldn't be needed.   The command_utils.py `class DaosCommand(ExecutableCommand)` is technically deprecated - replaced by `CommandWithSubCommand`.  I think it can be removed by now - there was a holdout at one point, but I forget which class still inherited it - and I don't see it used in the current master branch.   Be careful not to be confuse this with daos_utils.py `class DaosCommand(CommandWithSubCommand)`

Reply:
The removal of `def _get_result(self):` could also be applied to daos_utils.py `class DaosCommand(CommandWithSubCommand)`, but it could be done later too.

Reply:
Ah, sorry.  I see it now.  Not sure why I was blind to the `create_yaml()` call.

Everything looks good.